yarn-project/sequencer-client/src/block_builder/circuit_block_builder.test.ts

```diff
@@ -209,6 +230,37 @@ describe('sequencer/circuit_block_builder', () => {
       },
       10000,
     );
+
+    it('build edge case test', async () => {
+      // Regression test - this recreates the edge case
+
+      const simulator = await WasmCircuitSimulator.new();
+      const prover = new EmptyProver();
+      builder = new TestSubject(builderDb, vks, simulator, prover);
+      // update the starting tree
+      const updateVals = Array(16).fill(0n);
+      updateVals[0] = 19777494491628650244807463906174285795660759352776418619064841306523677458742n;
+      updateVals[1] = 10246291467305176436335175657884940686778521321101740385288169037814567547848n;
```

Comment:
Mind adding a comment on where these magic values came from? No need if it was just a random run, but if there's a sense to those values, let's explain it here.